tests/e2e/s3/02-init.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+unitTest: false
+commands:
+  - script: |
+      set -eux
+      export AWS_ACCESS_KEY_ID=minio
+      export AWS_SECRET_ACCESS_KEY=minio123
+      kubectl set env daemonset autoscaler-agent -n kube-system AWS_ACCESS_KEY_ID=minio
+      kubectl set env daemonset autoscaler-agent -n kube-system AWS_SECRET_ACCESS_KEY=minio123
+      
+      ../.venv/bin/python3 utils.py step2_init        
+      ../.venv/bin/python3 utils.py step2_assert
+      ../.venv/bin/python3 utils.py restore
```

Comment:
Is there a reason not to have `step2_assert` in a separate kuttl assertion step? I assumed that would naturally give the autoscaler-agent more time to generate the billing events.